hack/test-update-storage-objects.sh

```diff
@@ -112,8 +112,8 @@ tests=(
 examples/persistent-volume-provisioning/rbd/rbd-storage-class.yaml,storageclasses,,slow,v1beta1,v1
 )
 
-KUBE_OLD_API_VERSION="storage.k8s.io/v1beta1,extensions/v1beta1"
-KUBE_NEW_API_VERSION="storage.k8s.io/v1,extensions/v1beta1"
+KUBE_OLD_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1beta1,extensions/v1beta1"
+KUBE_NEW_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1,extensions/v1beta1"
```

Comment:
@sttts I think the parameter should be like this:
```bash
KUBE_OLD_API_VERSION="storage.k8s.io/v1beta1,extensions/v1beta1"
KUBE_NEW_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1"
KUBE_OLD_STORAGE_VERSIONS="extensions/v1beta1,storage.k8s.io/v1beta1"
# In step 3, networking objects will be converted from "extensions/v1beta1" and stored as "networking.k8s.io/v1".
# https://github.com/kubernetes/kubernetes/blob/v1.8.0-alpha.2/hack/test-update-storage-objects.sh#L175
KUBE_NEW_STORAGE_VERSIONS="networking.k8s.io/v1,storage.k8s.io/v1" 
```

> As we still write extensions/v1beta1 objects

Are you referring to fact with this PR, by default we still store objects as `extensions/v1beta1`? I agree. But we should test that we have the ability of converting the storage versions now.

Anyone, it doesn't block this PR.